org.freedesktop.Sdk.json.in

```diff
@@ -50,6 +50,12 @@
             "directory": "lib/sdk",
             "no-autodownload": true
         },
+        "org.freedesktop.Sdk.Tools": {
+            "directory": "bin/sdk",
+            "subdirectories": true,
+            "no-autodownload": true,
+            "autodelete": true
```

Comment:
I don't think setting autodelete is right. That means any runtime using this extension will remove all the tools when you uninstall it, but multiple runtimes may use these extensions.